src/actions/setIsMulticursorExecuting.ts

```diff
@@ -0,0 +1,29 @@
+import State from '../@types/State'
+import Thunk from '../@types/Thunk'
+
+interface SetIsMulticursorExecutingPayload {
+  value: boolean
+  /** Label for the undo operation (typically the command name). */
+  operationLabel?: string
```

Comment:
Let's call this `undoLabel` since it is only used that in that specific context.